neuralmonkey/encoders/transformer.py

```diff
@@ -27,7 +29,12 @@ def position_signal(dimension: int, length: tf.Tensor) -> tf.Tensor:
     positions = tf.to_float(tf.range(length))
 
     num_timescales = dimension // 2
-    log_timescale_increment = 4 / (tf.to_float(num_timescales) - 1)
+
+    # see: https://tinyurl.com/yck5etfo
+    magic = math.log(1.0e4 / 1.0)
```

Comment:
Co je tohle za blbost, `1.0e4 / 1.0` je `1.0e4` a `math.log(1.0e4)` je `4`, ne?

Reply:
`math.log(1.0e4) == math.log(10000) == 9.210340371976184`

Reply:
Děkuji.

Reply:
Souhlasim, ze staci math.log(1.0e4), oni to tam maji parametrizovane a ja jsem jen zkopiroval default hodnoty